AdvancedDLSupport.AOT.Tasks/AdvancedDLSupport.AOT.Tasks.targets

```diff
@@ -0,0 +1,15 @@
+<Project>
+  <PropertyGroup Condition="'$(AOTOutputDirectory)' == ''">
+    <AOTOutputDirectory>adl-aot</AOTOutputDirectory>
+  </PropertyGroup>
+  <Target Name="IncludeAOTAssemblies" BeforeTargets="BeforeBuild">
+    <ItemGroup>
+      <AdditionalFiles Include="$(AOTOutputDirectory)/*.dll" InProject="false" CopyToOutputDirectory="PreserveNewest" />
+    </ItemGroup>
+  </Target>
+  <Target Name="BuildAOTAssemblies" DependsOnTargets="Build">
+    <GenerateDLDynamicAssemblies Input="$(TargetPath)" Output="$(ProjectDir)/$(AOTOutputDirectory)" />
+    <Copy SourceFiles="$(ProjectDir)/$(AOTOutputDirectory)/*.dll"
```

Comment:
Would this leave files in the ProjectDir?

Reply:
Yes. Will change.

Reply:
Fixed.